backend/README.md

```diff
@@ -0,0 +1,43 @@
+## RUNNING THE BACKEND SERVER
+
+This README provides essential information for setting up and running the server.ts file in this project.
+
+### Prerequisites
+
+Before running the server.ts file, make sure you have the following prerequisites installed:
+
+-   **Node.js:** Ensure that you have Node.js installed on your machine. You can download it from [Node](https://nodejs.org/).
+
+-   **TypeScript:** Install TypeScript globally using the following command:
+
+    ```bash
+    npm install -g typescript
+    ```
+
+-   **TypeScript dependencies:** You can install them using the following command:
+
+    ```bash
+    npm install --save-dev typescript @types/node
+    ```
+
+-   **Project dependencies:** You can install them using the following command:
+
+    ```bash
+    npm install
+    ```
+
+### Configuration
+
+-   Create a .env file in the root of your project and define the following variable:
+
+    ```env
+    PORT=your_desired_port_number
```

Comment:
if env file is the root, .env should be included not in the .gitignore file in this folder, but in the root .gitignore file.

Reply:
Frontend now has its own .env file - .env added to .gitignore in the root.

Reply:
I don't see any of the new comments.